scripts/gitlab/deploy_artifact.sh

```diff
@@ -2,6 +2,6 @@
 
 set -ev
 
-vendor/bin/blt artifact:deploy --commit-msg "Automated commit by Gitlab Pipelines for Build $BUILD_BUILDID" --branch "$BUILD_SOURCEBRANCH-build" --no-interaction --verbose
+vendor/bin/blt artifact:deploy --commit-msg "Automated commit by Gitlab Pipelines for Build $CI_PIPELINE_IID" --branch "$CI_COMMIT_BRANCH-build" --no-interaction --verbose
 
```

Comment:
Replaced these with the [most relevant Gitlab predefined variables.](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html)